docs/openapi.json

```diff
@@ -13,25 +13,46 @@
     "version" : "1.0.0"
   },
   "paths" : {
-    "/contracts/v1/version" : {
-      "get" : {
-        "tags" : [ "version" ],
-        "summary" : "callable way of retrieving current version",
-        "description" : "callable way (compared to analyzing the URL of the endpoint) of retrieving current version. As well good for testing purposes as operation is get only",
-        "operationId" : "version",
+    "/contracts/v1/documents" : {
+      "post" : {
+        "tags" : [ "documents" ],
+        "summary" : "Upload a document for a contract.",
+        "description" : "Please note that this operation needs to be called per document for a contract to be uploaded. The response contains a handle to the document. This handle should be provided with the contract to be uploaded via uploadContract",
```

Comment:
I would use the perma linking to point to the call where the handle can be used.
E.g. https://stackoverflow.com/questions/52703804/how-to-link-to-another-endpoint-in-swagger